webapp/src/components/jira_ticket_tooltip/jira_ticket_tooltip.tsx

```diff
@@ -40,8 +40,12 @@ const myStatusClasses: Record<string, string> = {
 export default class TicketPopover extends React.PureComponent<Props, State> {
     constructor(props: Props) {
         super(props);
-        const {ticketID} = this.getIssueKey();
+        const issueKey = this.getIssueKey();
+        if (issueKey === null) {
```

Comment:
maybe this should work ?
```suggestion
        if (issueKey) {
```